go/pumiceDB/client/PumiceDBClient.go

```diff
@@ -75,14 +77,26 @@ func CToGoString(cstring *C.char) string {
 	return C.GoString(cstring)
 }
 
+// Getter method for request, response, and responseLen
+func (p *PmdbReqArgs) GetPmdbData() ([]byte, *[]byte, int64) {
+    return p.request, p.response, p.responseLen
+}
+
+// Setter method for request, response, and responseLen
+func (p *PmdbReqArgs) SetPmdbData(req []byte, resp *[]byte, len int64) {
+    p.request = req
+    p.response = resp
+    p.responseLen = len
+}
+
 //Get PumiceRequest in common format
 func getPmdbReq(reqArgs *PmdbReqArgs) (unsafe.Pointer, int64) {
```

Comment:
It looks like `getPmdbReq()` is no longer in use and can now be removed!

Reply:
Removed